ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>@title - @start_time</title>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+
+  <script type="text/javascript" src="js/zepto.min.js"></script>
+  <script type="text/javascript">
+    window.META_COLUMNS = 8;
+  </script>
+  <script type="text/javascript" src="js/setup.js?@(resource_hashes['setup.js'])"></script>
+  <script type="text/javascript">
+    var QUERY_TRANSFORMS = {
+      'id': '[.].*'
+    };
+  </script>
+
+  <link rel="stylesheet" type="text/css" href="css/status_page.css?@(resource_hashes['status_page.css'])" />
+  <link rel="stylesheet" type="text/css" href="css/blocked_releases_page.css?@(resource_hashes['blocked_releases_page.css'])" />
+</head>
+<body>
+  <script type="text/javascript">window.body_ready();</script>
+  <div class="top logo">
+    <h1><img src="http://wiki.ros.org/custom/images/ros_org.png" alt="ROS.org" width="150" height="32" /></h1>
+    <h2>Repos blocked by other repos<br />@(rosdistro_name)</h2>
+  </div>
+  <div class="top search">
+    <form action="?">
+      <input type="text" name="q" id="q" />
+      <p>
+        <a href="?q=" title="Show all repos">all</a>,
+        <a href="?q=_RELEASED" title="Repositories which have already been released">released</a>,
+        <a href="?q=_UNRELEASED" title="Repositories which have not been released">unreleased</a>,
+        <a href="?q=_BLOCKED" title="Repositories which are blocked from being released because of unreleased dependencies">blocked</a>,
+        <a href="?q=_UNBLOCKED" title="Repositories which can be released">releasable</a>,
+        <a href="?q=_UNBLOCKED_BLOCKING" title="Repositories which can be released and are preventing others from being released">releasable and blocking</a>,
+        <a href="?q=_UNBLOCKED_UNBLOCKING" title="Repositories which can be released and are not preventing others from being released">releasable and not blocking</a>,
+        <a href="?q=.metapackages" title="Repositories which are dependencies of the desktop-full metapackage">desktop-full</a>
```

Comment:
Got it. From brief inspection of the `setup.js` it looks like search strings will search across all columns by default and a regex is needed to filter by a column (e.g. repo name), is that correct? Maybe there should be a guide to the regex keys somewhere near the top of the page. Also sorry if this is outdated feedback based on the state of the site right now, I'm not generating it locally.